src/schema.rs

```diff
@@ -18,10 +18,24 @@ pub struct DetectSchema {
     pub version_files: Option<Vec<String>>,
 }
 
+#[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum HostLibc {
```

Comment:
Let me pull this over into the `system_env` crate, as it'll be useful there also.